### DISCUSSION TITLE
一直为未运行，然后报错，这种该怎么解决

### DISCUSSION THREAD

**Discussion by Magichcx** [messense/aliyundrive-webdav#566] [Q&A]
thread 'main' panicked at 'error binding to 192.168.2.110:8080: error creating server listener: Address not available (os error 99)', /root/.cargo/registry/src/github.com-1ecc6299db9ec823/hyper-0.14.20/src/server/server.rs:77:13
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

**Comment by stevenliou** [comment]
一样的问题，楼主解决了吗

**Comment by stevenliou** [comment]
找到原因了，我这边是因为把主机IP地址写错了，囧
